python/nav/ip.py

```diff
@@ -32,7 +32,7 @@ def __cmp__(self, other):
         try:
             return super(IP, self).__cmp__(other)
         except TypeError:
-            return cmp(self.ip, other)
+            return (self.ip > other) - (self.ip < other)
```

Comment:
I know this is the documented workaround but it is ugly and not Monday/sleepy/before caffeine compatible. If you need something that behaves like `cmp` more than once, making a function for it seems better for reading comprehension down the line.